python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py

```diff
@@ -133,6 +175,13 @@ def dyfunc(x):
     # in this case, unwraps it into a raw method or function.
     _, func = unwrap_decorators(func)
 
+    if getattr(func, DISABLE_CONVERTION, False):
+        translator_logger.log(
+            2,
+            "{} is not converted it is decorated by 'paddle.jit.not_to_static'.".
```

Comment:
`"{} is not converted when it is decorated by 'paddle.jit.not_to_static'."` add `when`

Reply:
Thanks, done